File: 1491.cpp
/*
* Given an array of unique integers salary where salary[i] is the salary of the employee i.
* Return the average salary of employees excluding the minimum and maximum salary.
*/
class Solution {
public:
    double average(vector<int>& salary) {
        int maxxy = INT_MIN;
        int minny = INT_MAX;
        double summy = 0;
        for(int v : salary){
            if(v > maxxy){maxxy = v;}
            if(v < minny){minny = v;}
            summy += (double) v;
        }
        return (summy - minny - maxxy) / (salary.size() - 2);
    }
};

File: problems/1491.py
"""
Given an array of unique integers salary where salary[i] is the salary of the employee i.

Return the average salary of employees excluding the minimum and maximum salary.
"""
class Solution:
    def average(self, salary: List[int]) -> float:
        summy, maxxy, minny = 0, 0, 1000000000
        for i in salary:
            if i > maxxy: maxxy = i
            if i < minny: minny = i
            summy += i
        return (summy - minny - maxxy) / (len(salary) - 2)